analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -20,8 +35,16 @@ public void DisposedTwice_Conditional()
         {
             d.Dispose();
         }
-        d.Dispose(); // FIXME Non-compliant {{Refactor this code to make sure 'd' is disposed only once.}}
-//          ^
+        d.Dispose(); // Noncompliant {{Resource 'd' has already been disposed explicitly or implicitly through a using statement. Please remove the redundant disposal.}}
+//      ^^^^^^^^^^^
+    }
+
+    public void DisposedTwice_AssignDisposableObjectToAnotherVariable()
```

Comment:
Same meaning, easier to read
```suggestion
    public void DisposedTwice_Relations()
```